pubtools/_quay/signature_handler.py

```diff
@@ -129,7 +129,7 @@ def create_manifest_claim_message(
         }
         return message
 
-    def get_tagged_image_digests(self, image_ref):
+    def get_tagged_image_digests(self, image_ref, media_type):
```

Comment:
This method doesn't take into account the existence of v2s1 at all. Will this even work?

Reply:
yeah, it doesn't work. Emily reported problem in this method in another JIRA